py-scripts/create_station.py

```diff
@@ -446,7 +458,12 @@ def build(self):
 
         logger.info("Creating stations")
         self.station_profile.set_command_flag("add_sta", "create_admin_down", 1)
-
+        if self.initial_band_pref.upper() not in ["2.4G", "2G", "5G", "6G"]:
```

Comment:
Not to add feature creep, but can you also support bands specified like `2.4GHz` and `5 ghz`? This will help better match the GUI.
![image](https://github.com/user-attachments/assets/69de54e1-8eaa-499c-b8b3-a943f8c7d95f)

Possibly one way to do this would be to accept any sensible band specification as input and then map it to one of `2.4GHz`, `5GHz`, or `6GHz` internally.